targets/TARGET_TOSHIBA/TARGET_TMPM46B/device/TOOLCHAIN_IAR/tmpm46bf10fg.icf

```diff
@@ -25,11 +25,11 @@ define region RAM_region   = mem:[from __ICFEDIT_region_RAM_start__   to __ICFED
 define block CSTACK    with alignment = 8, size = __ICFEDIT_size_cstack__   { };
 define block HEAP      with alignment = 8, size = __ICFEDIT_size_heap__     { };
 
-define block FLASH_CODE_ROM  {section FLASH_ROM_init object flash_api.o, section .text_init object tmpm46b_fc.o}; 
-define block FLASH_CODE_RAM  {section FLASH_ROM object flash_api.o, section .text object tmpm46b_fc.o};
+define block FLASH_CODE_ROM  {section FLASH_ROM_init object flash_api.o};
```

Comment:
what is this change about? removing some temporary object that should not be there? Separate commit for this change with reason for removing?

Reply:
This was modified after a the previous link failure in this board. Consulted with Toshiba engineers, and they said that this object file wasn't required there. Now, I can move this change to another PR, but it would only make sense in this one, as the link error was the only reason for this change.

Reply:
@0xc0170 Is this explanation satisfactory? If not, I will need to open another PR (with the reasoning behind it being the resolving of the link error in this one).

Reply:
OK Done. Now depending on [#7791](https://github.com/ARMmbed/mbed-os/pull/7791).